packages/jest/generators.ts

```diff
@@ -0,0 +1,2 @@
+export { jestInitGenerator } from './src/generators/init/init';
```

Comment:
Why is this file created?

Reply:
added it because inside the packages/jest/index.ts, it also exports a few utils functions that have imports from typescript. So everything import from `@nrwl/jest` directly, it would throw error like:

```
>  NX  Generating @nrwl/node:application
 >  NX   Cannot find module 'typescript'
   Require stack:
   - /private/tmp/nx-e2e/nx/proj7503815/node_modules/@nrwl/jest/src/utils/config/functions.js
   - /private/tmp/nx-e2e/nx/proj7503815/node_modules/@nrwl/jest/src/utils/config/update-config.js
   - /private/tmp/nx-e2e/nx/proj7503815/node_modules/@nrwl/jest/index.js
   - /private/tmp/nx-e2e/nx/proj7503815/node_modules/@nrwl/node/src/generators/init/init.js
 ```
so i added this generators.ts, so instead of doing an import `import { jestInitGenerator } from '@nrwl/jest';`, going to import from this generators: `import { jestInitGenerator } from '@nrwl/jest/generators';`